R/SQLiteDriver.R

```diff
@@ -21,7 +21,7 @@ setClass("SQLiteDriver",
 
 #' @rdname SQLiteDriver-class
 #' @export
-setMethod("dbDataType", "SQLiteDriver", function(dbObj, obj, ...) {
+setMethod("dbDataType", "SQLiteDriver", function(dbObj, obj, extended_types = FALSE, ...) {
```

Comment:
This forces naming the argument:

```suggestion
setMethod("dbDataType", "SQLiteDriver", function(dbObj, obj, ..., extended_types = FALSE) {
```